backend/app/api/api_v1/routers/lookups/__init__.py

```diff
@@ -0,0 +1,15 @@
+from .main import (  # noqa F401
+    lookup_geographies,
+    lookup_instruments,
+    lookup_languages,
+    lookup_sources,
+    lookup_document_categories,
+    lookup_document_types,
+    lookup_sectors,
+)
+
+from .geo_stats import lookup_geo_stats  # noqa F401
+
+from .main import tree_table_to_json  # noqa F401
```

Comment:
is there any reason to import these utility functions here? are they used elsewhere?

Reply:
I've removed this and explicitly added the import into the tests that were using it.